### PR TITLE
Release 0.5.0

### PR DESCRIPTION
**Pull request by katstevens**
## 0.5.0

🆕 New features:

New component: Cookie banner and analytics

  *Requirement before using this component:*
  - To use this the application must be using `govuk-frontend` template
  - There must be functionality for the user to change their cookie preferences

  *Installing component:*
  1. Check config.py has this line for jinja to find the components
      ```
      os.path.join(repo_root, 'node_modules', 'digitalmarketplace-govuk-frontend')
      ```
  2. Import the component in `_base_page.html`
     ```
     {% from "digitalmarketplace/components/cookie-banner/macro.njk" import dmCookieBanner %}
     ```
  3. Include the component `{{ dmCookieBanner({...}) }}` in the `{% block header %}` block.
     For its parameters, see its README and/or YAML. For examples, see the review app.
  ([PR #49](https://github.com/alphagov/digitalmarketplace-govuk-frontend/pull/49))

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -32,6 +32,8 @@
 
   ([PR #58](https://github.com/alphagov/digitalmarketplace-govuk-frontend/pull/58))
 
+## 0.5.0
+
```
Comment (lfdebrux) [inline CHANGELOG.md]: This release also includes the fix above 😁

Comment (katstevens) [reply CHANGELOG.md]: OK, I would have never spotted that. Please can we do something to make the actual changes distinct from the template?

Comment (lfdebrux) [reply CHANGELOG.md]: Good idea... how? 🤔

Comment (katstevens) [reply CHANGELOG.md]: I've added a commit.

**Pull request merged**